### DISCUSSION TITLE
Waymo converted to KITTI format.

### DISCUSSION THREAD

**Discussion by Xiangxu-0103** [open-mmlab/mmdetection3d#2095] [General] [answered]
PR #2065 may cause an error when `converting  waymo dataset to kitti format`.

Error message:

> tensorflow.python.framework.errors_impl.InternalError: CUDA runtime implicit initialization on GPU:0 failed. Status: out of memory

When decreasing `num_workers` to 64, the error can be:

> tensorflow.python.framework.errors_impl.InternalError: Attempting to perform BLAS operation using StreamExecutor without BLAS support [Op:BatchMatMulV2]

When removing the lines added in PR #2065, everything is ok.

**Comment by ZCMax** [comment, marked as answer]
Have you encountered the  stuck in torch.einsum when create dataset problem after removing the lines added in PR #2065

**Reply by Xiangxu-0103** [reply]
I didn't encounter the stuck problem.
